test/js/blackbox_video_export_guard.test.js

```
@@ -0,0 +1,56 @@
+import { afterEach, describe, expect, it, vi } from "vitest";
+import {
+    animationLoop,
```

ESLint

[error] 'animationLoop' is defined but never used.

(unused-imports/no-unused-imports)